### PR TITLE
Add Compression Standard to db.json

### PR DESCRIPTION
**Pull request by saschanaz**
Corresponding to https://github.com/whatwg/sg/issues/231.

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -441,6 +441,27 @@
...
+            },
+            {
+              "name": "Canon Mukai",
+              "email": "canonmukai@google.com"
+            }
+          ],
+          "reference": "COMPRESSION",
+          "review_draft_schedule": [
+            2,
+            8
```
Comment (annevk) [inline db.json:461]:
```suggestion
            4,
            10
```

File: db.json [hunk 2]
```diff
@@ -441,6 +441,27 @@
...
+          ],
+          "twitter": "compressionstandard"
```
Comment (annevk) [inline db.json:463]: @domenic can you set this up?

**Review by annevk — COMMENTED** [review]
@domenic looks like you are in charge of the Workstream. Happy with all these deputy editors?

@saschanaz should you be an editor as well?

File: db.json (hunk 1 above)
Comment (saschanaz) [reply db.json:461]: I'm merging this, but I have no idea how is this decided.

**Comment by saschanaz** [comment]
> should you be an editor as well?

I'm only helping this on behalf of @ricea and not really an editor. I'm open to the position though 😄

**Comment by domenic** [comment]
> @domenic looks like you are in charge of the Workstream. Happy with all these deputy editors?

Yes, seems good!

**Comment by ricea** [comment]
It might be good to replace Canon Mukai with a different editor as she doesn't work on web standards nowadays. I will ask her.

**Comment by domenic** [comment]
Yeah, we seem to use the "authors" field as a mix of historical authors and current deputy editors. (The Streams Standard, HTML Standard, and URL Pattern Standards notably have some historical authors still listed.)

**Comment by ricea** [comment]
So maybe Canon should remain as an author but not as an editor? She said she was happy to be moved to "former editors".

**Comment by annevk** [comment]
I'm not sure we're consistent about that. E.g., I'm not listed for Fullscreen. I think I'd prefer it being a reflection of current editors as otherwise it would get quite unwieldy over a long period of time.

**Review by annevk — APPROVED** [review]
I've moved Canon to the Acknowledgments section of the standard. The Twitter account isn't live yet, but since the standard is I'm going to merge this.

Thanks everyone!

**Pull request merged**